Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+
+It is therefore not surprising that the visual interaction of concepts installs them deep into our minds, almost experientially, rather than conceptually!
+
+Imagine, for example, thinking about the formula of the diagonal of a square... reading it is a very abstract thing, but if we imagine in a _visual_ way the graphical reason that leads to the formula, it will make much more sense!
+
+Now, fortunately, there are incredibly skilled and intelligent people in the world, and some of them have created this:
+
+[**Immersive math - Linear Algebra**](http://immersivemath.com/ila/index.html)
+
+Once you click on the link and start exploring this interactive book, you won't believe your eyes for the clarity and effectiveness with which the concepts are explained. 
+
+This book currently contains so much information and so well done that it deserves a front-row seat in Virgilio!
+
+Take your time, explore the book, fall in love with those interactive explanations!
+
+Going through this would be sufficient to get all the remaining knowledge you need from a mathematical perspective, to understand most of the papers and code out there!
+
+Note:
+Chapter 6 is about Matrix Algebra, and it's indeed very good (like the rest of the book), but we're **very proud** of [**_Virgilio's official Matrix Algebra guide_**](https://github.com/virgili0/Virgilio/blob/master/NewToDataScience/MatrixAlgebra.ipynb), and we suggest you use both resources! 
+
+Moreover, Virgilio's guide to Matrix Algebra is an _interactive Jupyter Notebook_, with exercises included!
+
+We suggest to go through this latter resource once (or if) you are already practicing with the [Python](https://www.python.org/) programming language. 
+
+#### Check your knowledge
+
+Test your knowledge with [**these**](https://web.pdx.edu/~erdman/LINALG/Linalg_pdf.pdf) exercises.
+
+Anyway you can find a ton of them online, and remember, that _the more you make exercise, the better!_
+
+### Optimization
+
+Calculus, Linear Algebra, and Matrix Algebra are the 3 fundamental pillars you need to be confident with in order to understand most of the mathematical implications of Machine Learning systems.
+
+This is important in Data Science because developing Machine Learning predictive models play a big role during a Data Science project, and these models are often the most useful outcome!
+
+The **optimization course** is a bit more advanced, so maybe you could not tackle _during the first pass of this guide_, but later be sure to take the following lectures, from the professor [Geoff Gordon](https://www.youtube.com/channel/UC4XBoaEXuvob0mQraQJ7wNw):
+
+[**Introduction to Convex Optimization**](https://www.youtube.com/playlist?list=PL7y-1rk2cCsDOv91McLOnV4kExFfTB7dU)
+
+Moreover, if you want to dive deeper (very deep), you can hold this close to your bed:
```

Comment:
> you can hold this close to your bed:

or 

> we suggest this for bed-time reading

Reply:
I prefer the former.

Reply:
Ok, sure again, might not be a way of saying it.

Reply:
ok this sounds bad I change it